collect_app/src/test/java/org/odk/collect/android/entities/DatabaseEntitiesRepositoryTest.kt

```diff
@@ -36,4 +37,19 @@ class DatabaseEntitiesRepositoryTest : EntitiesRepositoryTest() {
         repository.save("things", savedEntity)
         assertThat(repository.getEntities("things")[0], sameEntityAs(savedEntity))
     }
+
+    @Test
+    fun `#save ignores case-insensitive duplicate properties`() {
```

Comment:
From the conversation on Slack, it looks like we want this to be a feature of entities in general, so I think this should move to `EntitiesRepostitoryTest`.

Reply:
Done.